linera-service/src/linera.rs

```diff
@@ -894,6 +895,36 @@ enum ProjectCommand {
     ///
     /// Equivalent to running `cargo test` with the appropriate test runner.
     Test { path: Option<PathBuf> },
+    /// Build and publish a Linera project.
+    PublishAndCreate {
+        /// The path of the root of the Linera project.
+        /// Defaults to CWD if unspecified.
+        path: Option<PathBuf>,
+
+        /// An optional chain ID to publish the bytecode. The default chain of the wallet
+        /// is used otherwise.
+        publisher: Option<ChainId>,
+
+        /// The shared parameters as JSON string.
+        #[structopt(long)]
+        json_parameters: Option<String>,
+
+        /// Path to a JSON file containing the shared parameters.
+        #[structopt(long)]
+        json_parameters_path: Option<PathBuf>,
+
+        /// The initialization argument as a JSON string.
+        #[structopt(long)]
+        json_argument: Option<String>,
+
+        /// Path to a JSON file containing the initialization argument.
+        #[structopt(long)]
+        json_argument_path: Option<PathBuf>,
```

Comment:
Should we rename these to `json_initialization_argument[_path]`? (Also needs docs update, of course, so maybe a separate PR.)

Reply:
I'll add a ticket for this :) Probably should be in a separate PR.